### PR TITLE
Add optimistic label events

### LINKED ISSUE #971: Labeled event on optimistic mutation
<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.13.0 (1191)
Device: iPhone X (iOS 11.1.1)
TestFlight: false
</pre>
</details>

### PR DESCRIPTION
**Pull request by rnystrom**
Fixes #971

### CODE REVIEW

File: Classes/Issues/GithubClient+Issues.swift [hunk 1]
```diff
@@ -334,7 +334,41 @@ extension GithubClient {
...
         labels: [RepositoryLabel]
         ) {
-        let optimistic = previous.updated(labels: IssueLabelsModel(labels: labels))
+        guard let actor = userSession?.username else { return }
+
+        let oldLabelNames = Set<String>(previous.labels.labels.map { $0.name })
+        let newLabelNames = Set<String>(labels.map { $0.name })
+
+        var newEvents = [IssueLabeledModel]()
+        for newLabel in labels {
```
Comment (BasThomas) [inline Classes/Issues/GithubClient+Issues.swift]: `filter` -> `forEach`?

File: Classes/Issues/GithubClient+Issues.swift [hunk 2]
```diff
@@ -334,7 +334,41 @@ extension GithubClient {
...
+                    id: UUID().uuidString,
+                    actor: actor,
+                    title: newLabel.name,
+                    color: newLabel.color,
+                    date: Date(),
+                    type: .added
+                ))
+            }
+        }
+        for oldLabel in previous.labels.labels {
```
Comment (BasThomas) [inline Classes/Issues/GithubClient+Issues.swift]: Same, `filter` -> `forEach`?

File: Classes/Issues/GithubClient+Issues.swift (hunk 1 above)
Comment (rnystrom) [reply Classes/Issues/GithubClient+Issues.swift]: That’s 2x the iterations tho, right?

Comment (BasThomas) [reply Classes/Issues/GithubClient+Issues.swift]: Ah yeah, good point 👍

**Pull request merged**